ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.Data.Abstractions.Repositories
+{
+    public interface IUserRepository
+    {
+        Task<UserDTO> GetUser(string identity);
+        Task<IEnumerable<UserDTO>> GetUsers();
+        Task<IEnumerable<string>> GetUserRoles(string identity);
```

Comment:
But what about this method which is not used anymore?

Reply:
Method has been removed